nvm.sh

```diff
@@ -1848,13 +1848,14 @@ nvm_install_source() {
 
   local make
   make='make'
-  if [ "${NVM_OS}" = 'freebsd' ]; then
-    make='gmake'
-    MAKE_CXX='CXX=c++'
-  elif [ "${NVM_OS}" = 'aix' ]; then
+  if [ "${NVM_OS}" = 'freebsd' ] || [ "${NVM_OS}" = 'aix' ]; then
     make='gmake'
   fi
-  if nvm_has "clang++" && nvm_has "clang" && nvm_version_greater_than_or_equal_to nvm_clang_version 3.5 ; then
+  if [ "${NVM_OS}" = 'freebsd' ] || [ "${NVM_OS}" = 'darwin' ]; then
+    [ -z "${CXX-}" ] && MAKE_CXX='CC=cc CXX=c++'
```

Comment:
`nvm` has always blindly overridden `CC` and `CXX` before, and i've never once seen a request to override those things. Why is now different?